docs/protocol/reference/deployed.md

```diff
@@ -32,45 +32,31 @@ Rinkeby (LPT) is the set of contracts that originally implemented the alpha prot
-- ServiceRegistry (Target): [0x65594869bB21858428c16629838940c273fe25ce](https://rinkeby.etherscan.io/address/0x65594869bb21858428c16629838940c273fe25ce)
-- LivepeerTokenFaucet: [0x6b752dA302C80795ACc9347AF8f472D3473b19f2](https://rinkeby.etherscan.io/address/0x6b752dA302C80795ACc9347AF8f472D3473b19f2)
-- SortedDoublyLL: [0xdB7f48a3e6fC6caCD646CE43C1aF48CcDFa78C60](https://rinkeby.etherscan.io/address/0xdB7f48a3e6fC6caCD646CE43C1aF48CcDFa78C60)
-- PollCreator: [0x149805EF90FaDA12D27e8a020b6c138F3d86A9a3](https://rinkeby.etherscan.io/address/0x149805ef90fada12d27e8a020b6c138f3d86a9a3)
-- MerkleSnapshot: [0x0dC152D2BfA9d213cD449F52244e1080e8F7Ad31](https://rinkeby.etherscan.io/address/0x0dC152D2BfA9d213cD449F52244e1080e8F7Ad31)
-
-## Rinkeby (LPT)
-- Controller: [0x37dC71366Ec655093b9930bc816E16e6b587F968](https://rinkeby.etherscan.io/address/0x37dC71366Ec655093b9930bc816E16e6b587F968)
-- LivepeerToken: [0x24f56eF2fED379421BEE64Ecafc2bC744f72522b](https://rinkeby.etherscan.io/address/0x24f56eF2fED379421BEE64Ecafc2bC744f72522b)
-- Minter: [0xbbce46C5C018497f13197024171Fd442d9EF5a25](https://rinkeby.etherscan.io/address/0xbbce46C5C018497f13197024171Fd442d9EF5a25)
-- BondingManager (Proxy): [0xF6b0Ceb5e3f25b6FBecf8186F8A68B4E42A96a17](https://rinkeby.etherscan.io/address/0xF6b0Ceb5e3f25b6FBecf8186F8A68B4E42A96a17)
-- BondingManager (Target): [0x60C393F25a1533E28E37f8e875D1BB5ea929B5B2](https://rinkeby.etherscan.io/address/0x60C393F25a1533E28E37f8e875D1BB5ea929B5B2)
-- TicketBroker (Proxy): [0xA76D4ef7D4B6611987a9a605B71C6ffAaA8190c4](https://rinkeby.etherscan.io/address/0xa76d4ef7d4b6611987a9a605b71c6ffaaa8190c4)
-- TicketBroker (Target): [0xbBDE3c2A5813759bA1c92E2e31b55cE98f4a1f5A](https://rinkeby.etherscan.io/address/0xbbde3c2a5813759ba1c92e2e31b55ce98f4a1f5a)
-- RoundsManager (Proxy): [0xEB0EF46B5771D523402234FF0d7596d2C62411dE](https://rinkeby.etherscan.io/address/0xEB0EF46B5771D523402234FF0d7596d2C62411dE)
-- RoundsManager (Target): [0xaF59230530fad87Db90C4280D62ebb64c357b98f](https://rinkeby.etherscan.io/address/0xaf59230530fad87db90c4280d62ebb64c357b98f)
-- ServiceRegistry (Proxy): [0x21b1E9dcFf801f71C3e5c7385FD16d0742cb64D1](https://rinkeby.etherscan.io/address/0x21b1e9dcff801f71c3e5c7385fd16d0742cb64d1)
-- ServiceRegistry (Target): [0x2f99B275E0581df2859a10Eb2DB907eED163aEaA](https://rinkeby.etherscan.io/address/0x2f99b275e0581df2859a10eb2db907eed163aeaa)
-- LivepeerTokenFaucet: [0xD69A2eC8f2F83267CE82A6d01837045BaE0EB9Bc](https://rinkeby.etherscan.io/address/0xD69A2eC8f2F83267CE82A6d01837045BaE0EB9Bc)
-- SortedDoublyLL: [0xD9Fff7E9c61648B16c6FB24b0119841221Bcfd3E](https://rinkeby.etherscan.io/address/0xd9fff7e9c61648b16c6fb24b0119841221bcfd3e)
-- Refunder: [0xD98a4e722274936B2D0Ceee71CB4eD30A67Ae48C](https://rinkeby.etherscan.io/address/0xd98a4e722274936b2d0ceee71cb4ed30a67ae48c)
 
 
 # Historical (DEPRECATED - FOR REFERENCE ONLY)
 
-## Mainnet (Deprecated)
+## Ethereum Mainnet (Deprecated)
```

Comment:
If everything under the above header is deprecated we could save some text and drop the "(Deprecated)" here and in the other sections below.